modules/nf-core/cellranger/vdj/main.nf

```diff
@@ -0,0 +1,54 @@
+process CELLRANGER_VDJ {
+    tag "${meta.id}"
+    label 'process_high'
+
+    container "nfcore/cellranger:7.1.0"
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "CELLRANGER_VDJ module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    input:
+    tuple val(meta), path(reads)
+    path  reference
+
+    output:
+    tuple val(meta), path("${meta.id}/outs/*"), emit: outs
+    path "versions.yml"                       , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def sample_arg = meta.samples.unique().join(",")
```

Comment:
Should be removed, the `--sample` flag can be presumably supplied directly via `ext.args` with wahtever meta field the pipeline deve may want to call it.

Reply:
note that this will affect downstream pipelines like [nf-core/scrnaseq(https://github.com/nf-core/scrnaseq/blob/c6ec2175a8f5d593c00136835c45c6291821fe11/subworkflows/local/align_cellranger.nf#L37)

however, in that case, `meta.samples` is simply a copy of `meta.id`, so it's probably safe to remove

to preclude `cellranger` loading extraneous FASTQs in the local directory (from `--fastqs=.`), the wisest change is probably to use `--sample=$meta.id`

that will ensure that Cellranger loads only the FASTQs relevant to the current sample in `meta.id`

Reply:
That's fine - the pipeline needs to serve the module in a way (As the modules are what are standardised)!